lib/kafo/configuration.rb

```diff
@@ -264,7 +264,7 @@ def migrations_dir
 
     def parser_cache
       if app[:parser_cache_path]
-        @parser_cache ||= Kafo::ParserCacheReader.new_from_file(File.expand_path(app[:parser_cache_path]))
+        @parser_cache ||= Kafo::ParserCacheReader.new_from_file(app[:parser_cache_path])
```

Comment:
Since we are removing `File.expand_path` here, we should probably add it into `#new_from_file` mapped to each path that ParserCacheReader is being handed to still account for relative paths.